service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationJobFlightMapKeys.java

```diff
@@ -8,6 +8,8 @@ public abstract class RunImputationJobFlightMapKeys {
   public static final String ALL_PIPELINE_INPUTS = "all_pipeline_inputs";
   public static final String CONTROL_WORKSPACE_STORAGE_CONTAINER_NAME =
       "control_workspace_storage_container_name";
+  public static final String CONTROL_WORKSPACE_STORAGE_CONTAINER_PROTOCOL =
+      "control_workspace_storage_container_protocol";
```

Comment:
i wonder what the limit of these strings are in the stairway database hahaha

Reply:
haha yeah we apparently haven't hit the limit yet. want me to find out?